woocommerce-services.php

```diff
@@ -1778,16 +1797,24 @@ public function render_schema_notices() {
 			}
 		}
 
+		/**
+		 * Check if WooCommerce Shipping has been activated.
+		 *
+		 * @return bool
+		 */
+		public function is_wc_shipping_activated() {
+			return in_array( 'woocommerce-shipping/woocommerce-shipping.php', get_option( 'active_plugins' ) );
+		}
+
 		/**
 		 * Returns if both Woo Shipping and Woo Tax are active.
 		 *
 		 * @return bool
 		 */
 		public function are_woo_shipping_and_woo_tax_active() {
```

Comment:
Do we still need this function?

Reply:
Yup! You can read a bit more about the case we need to support here: https://github.com/woocommerce/woocommerce-shipping/pull/586

TL;DR: The only WC Tax support we have right now in WCS&T is when both WC Tax and WC Shipping is activated, so we should keep this around to maintain some sort of backwards compatibility for WC Tax 😄